templates/documents/general_list.html

```diff
@@ -2,13 +2,38 @@
 <head>
 {php}html_header_show();{/php}
 {php}$this->assign('GLOBALS', $GLOBALS);{/php}
-
+{php}
+$place_hld = isset($_GET['patient_name'])?$_GET['patient_name']:xl("Patient search or select.");
+$cur_pid = isset($_GET['patient_id'])?$_GET['patient_id']:'';
+if ( isset($_SESSION['isNew']) && !isset($_GET['patient_name']) && isset($_GET['list']) && (int)$cur_pid != 0) {
+    unset($_SESSION['isNew']);
+}
+elseif ($cur_pid == '00') {
+    $_SESSION['isNew'] = true;
+    $cur_pid = '0';
```

Comment:
There is an odd thing that happens if have both document screens(the patient specific and the one in Miscellaneous) open in separate tabs. In that case it appears the isNew session causes the selector to show up in the patient specific documents screen which seems odd (if I am in a patient's chart then shouldn't be able to seek other patient's).